web/app.js

```diff
@@ -2029,6 +2044,15 @@ const PDFViewerApplication = {
       _windowAbortController: { signal },
     } = this;
 
+    this._touchManager = new TouchManager({
+      container: window,
+      overlayManager: this.overlayManager,
+      isPinchingDisabled: () => this.pdfViewer.isInPresentationMode,
+      onPinching: this.touchPinchCallback.bind(this),
+      onPinchEnd: this.touchPinchEndCallback.bind(this),
+      signal: this._windowAbortController.signal,
```

Comment:
I believe this is possible to shorten now.
```suggestion
      signal,
```